roles/container-engine/containerd/templates/config.toml.j2

```diff
@@ -8,7 +8,9 @@ oom_score = {{ containerd_oom_score }}
   max_send_message_size = {{ containerd_grpc_max_send_message_size }}
 
 [debug]
+  address = "{{ containerd_debug_address }}"
   level = "{{ containerd_debug_level }}"
+  format = "{{ containerd_debug_format }}"
 
```

Comment:
HI @spnngl 

When using `containerd config default`, it outputs:
```
[debug]
  address = ""
  format = ""
  gid = 0
  level = ""
  uid = 0
```

So, default config should not be changed :-)

If we want to customize the config, we can be using:
 
```
{% if containerd_debug_address %}
  address = "{{ containerd_debug_address }}"
{% else %}
```


The containerd debug is not "required config"

Thanks

Reply:
Changed it in last push

Reply:
Hum, I really disagree with that. It's of little use to match the output of `containerd config default`, because it's not like we can use that directly in our jinja template, so we have to update mostly manually anyway if containerd defaults change.

I'd rather have explicit defaults and a more straightforward template, with as little branching as possible.

Reply:
I put `containerd_debug_level` back to its previous value and the others will be used only if defined by the user

Reply:
What I meant is that we should not have to use `is defined` everywhere, and keeping the same structure that the output of `containerd config default` buy us nothing, so we should just have something like:
```yaml
{% if containerd_debug_enabled %}
[debug]
  setting_1 = {{ containerd_default_value_for_setting_1 }}
  ...
{% endif %}
```